### PR TITLE
Skripchenkoveta hometask 4

### PR DESCRIPTION
**Pull request by skripchenkoVeta**
Benchmark for task 4.2 (Hash)
Снимок экрана от 2021-11-26 23-11-11
Снимок экрана от 2021-11-26 23-21-21

Benchmark for task 4.1 (Array)
rotate and lab_rotate
Снимок экрана от 2021-11-26 23-12-41
rotate! and lab_rotate!
Снимок экрана от 2021-11-26 23-13-01

### CODE REVIEW

**Review by jtethras — CHANGES_REQUESTED** [review]
Both Array extensions and custom Hash are not implemented correctly. Let's get back to this task once you're ready!

**Comment by skripchenkoVeta** [comment]
okay, thanks

File: skripchenkoveta/hometask-4/my_hash.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    pair = get(index, key)
+    if pair
+      pair[1] = value
+    else
+      @count += 1
+      @myhash[index].push([key, value])
+    end
+  end
+
+  def get(index, key)
```
Comment (jtethras) [inline skripchenkoveta/hometask-4/my_hash.rb:35]: 🥇

**Review by jtethras — APPROVED** [review]
Great work! Love the new benchmarks and updates on the array extensions!

**Pull request merged**